src/applications/vaos/covid-19-vaccine/index.jsx

```diff
@@ -176,16 +175,6 @@ export function NewBookingSection() {
         <Route path={`${match.url}/review`}>
           <ReviewPage changeCrumb={newTitle => setCrumb(newTitle)} />
         </Route>
-        <Route
-          exact
-          path={
-            featureBreadcrumbUrlUpdate
-              ? `${match.path}/:id`
```

Comment:
FYI @ryanshaw this will likely conflict with your breadcrumbs PR. I assume it'll be fine since your PR will likely go in first but I thought I'd let you know anyways.

Also, for additional context, the [scheduling confirmation logic](https://github.com/department-of-veterans-affairs/vets-website/blob/340423fe53ba70623734ac8c0c7924fc730714c6/src/applications/vaos/covid-19-vaccine/redux/actions.js#L435) routes to `my-health/appointments/:id` not `my-health/appointments/covid-19-vaccine/:id` so it's safe to remove this route.